### ISSUE TITLE
mobilenet_v3中的InvertedResidual

### ISSUE THREAD

**Issue by wanganding** [bubbliiiing/mobilenet-yolov4-lite-pytorch#16]
# pw
                nn.Conv2d(inp, hidden_dim, 1, 1, 0, bias=False),
                nn.BatchNorm2d(hidden_dim),
                h_swish() if use_hs else nn.ReLU(inplace=True),

                # dw
                nn.Conv2d(hidden_dim, hidden_dim, kernel_size, stride, (kernel_size - 1) // 2, groups=hidden_dim, bias=False),
                nn.BatchNorm2d(hidden_dim),

                # Squeeze-and-Excite
                SELayer(hidden_dim) if use_se else nn.Identity(),

                h_swish() if use_hs else nn.ReLU(inplace=True),

                # pw-linear
                nn.Conv2d(hidden_dim, oup, 1, 1, 0, bias=False),
                nn.BatchNorm2d(oup),

Squeeze-and-Excite之后的激活函数是不是应该放在SElayer之前

**Comment by bubbliiiing** [comment]
啊哼？什么意思

**Comment by wanganding** [comment]
激活函数的位置不影响吗
